CHANGELOG.md

```diff
@@ -12,13 +12,19 @@
 
 - Downgraded the level of several code generator logs from `Info` to `Trace`. [#1277](https://github.com/spatialos/gdk-for-unity/pull/1277)
 
+### Fixed
+
+- Fixed a bug where the `ImprobableCodegen.marker` file would not get created despite the code generator returning successfully. [#1294](https://github.com/spatialos/gdk-for-unity/pull/1294)
```

Comment:
What would a user typically see when affected by this bug? 

The fact that its related to the marker file is sort of by-the-by

Reply:
how about
```suggestion
- Fixed a bug where the code generation would happen on each Unity compilation, despite the code generator returning successfully. [#1294](https://github.com/spatialos/gdk-for-unity/pull/1294)
```

Reply:
Yeah looks good!